products/jbrowse-react-linear-genome-view/stories/GettingStarted.stories.mdx

```diff
@@ -172,9 +165,7 @@ const state = createViewState({
 
 Example
 
-<Canvas>
-  <Story id="linear-view--using-loc-object" name="Using loc object" />
-</Canvas>
+<Story id="linear-view--using-loc-object" name="Using loc object" />
 
 ## Additional resources
```

Comment:
the guide for creating default sessions has a broken link now?

maybe clarify that "code example" is "Code example embedding react-linear-genome-view into a React page"..or something

Reply:
fixed link for default sessions, added your clarification text, and also added link out to https://jbrowse.org/jb2/docs/embedded_components/